### ISSUE TITLE
Deprecate and remove the old calendar views in favor of FullCalendar

### ISSUE THREAD

**Issue by jdufresne** [llazzaro/django-scheduler#378]
I'd like to propose that django-scheduler begins deprecating and removing the old views in favor the new FullCalendar views. It seems like an unnecessary amount of extra maintenance burden to maintain two sets of frontends. IMO, the FullCalendar work was a very nice improvement and so I suggest focusing future work in that direction. Removing the extra views should help reduce the complexity of the code base resulting in fewer bugs and easier maintenance of the better supported views.

Opening this as an issue first to get feedback on the suggestion. I'm happy to start putting in the work if it is agreeable.

**Comment by josassy** [comment]
I agree that this is a good option. I am newer to using this project, but the current inclusion of the old HTML and CSS templates caused me to waste time digging around for how to use them before I realized that the project had moved toward using the API with FullCalendar, etc.

Technically speaking, this doesn't seem like a difficult change to implement. Is there a reason this hasn't happened yet?

**Comment by llazzaro** [comment]
I agree on this change also, but I don't have time to do it. Any pull request will be welcome!
